css/Style.css

```diff
@@ -1 +1,113 @@
-li { color: blue; }
+html { box-sizing: border-box; }
+
+#header-background {
+  display: flex;
+  width: 375px;
+  height: 718px;
+  background-image: url("../img/header-shapes\ mobile.svg");
+  background-repeat: no-repeat;
+  flex-direction: column;
+
+  /* align-items: center; */
+}
+
+#header-background > * {
+  flex-grow: 0.5;
+}
+
+/* #header-background #primary-text {
+    flex: 1;
+} */
+
+#flex-nav {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  width: 375px;
+  height: 48px;
+}
+
+#flex-nav ul {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+}
+
+#burguer-menu-icon {
+  width: 24px;
+  height: 24px;
+  top: 8px;
+  left: 8px;
+
+  /* flex: 1; */
+  background-color: white;
+  position: absolute;
+}
+
+#logo {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+}
```

Comment:
- [ ] [OPTIONAL] Great job 👍 . Kindly use the correct family as seen in your chosen `Figma` design 🙏